sql/updates/world/3.3.5/9999_99_99_99_world_335.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM `spell_script_names` WHERE `spell_id`= 52267;
```

Comment:
Always delete by script name, never with just id.

Reply:
What if deleting the `ScriptName` causes 12 other spells to fail?

Reply:
scriptname + id

Reply:
⏫